src/styles/base.scss

```diff
@@ -0,0 +1,59 @@
+body {
+  margin: 0;
+  --column-count: 1;
+}
+
+
+
+.main {
+  padding-top: 50px;
+  padding-inline: 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), $card-width);
```

Comment:
The variable `$card-width` is used here, but it is not defined in this file. Ensure that `$card-width` is defined somewhere in your SCSS files before it is used.